server/routes/appointments/create-and-edit/appointmentJourney.ts

```diff
@@ -50,4 +50,6 @@ export type AppointmentJourney = {
   repeatPeriod?: AppointmentRepeatPeriod
   repeatCount?: number
   comment?: string
+
+  prisonNumber?: string
```

Comment:
Not a fan of this `prisonNumber` property. I think using the existing `prisoners` array with a `source` enum for the backlink would be a better approach personally. The use of this property is a little ambiguous.

Reply:
I think I agree with this point however the enum change is larger. Is a compromise to rename this property `fromPrisonNumberProfile` to make it explicit what the property is for? If this needs wider discussion, let's move it to Slack

Reply:
Changed property name as suggested